### ISSUE TITLE
client.ts not loading in an iframe

### ISSUE THREAD

**Issue by juanfran** [Shakeskeyboarde/vite-live-preview#2]
I am unable to get the functionality to work when the page is in an iframe. I'm getting the following error:

```shell
Access to script at 'http://localhost:4400/vite-live-preview/client.ts' from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

The regular Vite livereload works as expected. The problem seems to be caused by the `crossorigin` attribute:

```html
content += `<script crossorigin="" src=${clientSrc}></script>`;
```

Please let me know if you need a repository for reproduction.

**Comment by Shakeskeyboarde** [comment]
Interesting. I'll take a look. A repo would save me some time in trying to reproduce the problem.

**Comment by juanfran** [comment]
Here is the link to the repository https://github.com/juanfran/vl-preview-issue. While I was working on the repository, I noticed that the issue occurs because I'm using an iframe in sandbox mode. However, the regular Vite server works fine.

**Comment by Shakeskeyboarde** [comment]
Yup, `crossorigin` was the problem and unnecessary for a dev server. Fixed in `0.2.2`.
